src/makeRobot.js

```diff
@@ -27,18 +29,90 @@
  *
  * @typedef {object} Robot
  * @property {string} name
- * @property {number} wheels
- * @property {number} version
+ * @property {distanceber} wheels
+ * @property {distanceber} version
  * @property {function} info
  *
  * @param {string} name
- * @param {number} wheels
- * @param {number} version
+ * @param {distanceber} wheels
+ * @param {distanceber} version
  *
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${
+        this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(distance) {
+      if (distance === undefined) {
+        this.coords.y++;
```

Comment:
```suggestion
    goForward(distance = 1) {
```

Краще просто задати дефолтне значення в одиницю, ніж робити дві перевірки

Reply:
Точно! Классная идея!